packages/bpk-component-breadcrumb/src/BpkBreadcrumbItem.module.scss

```diff
@@ -19,6 +19,11 @@
 @use '../../unstable__bpk-mixins/tokens';
 @use '../../unstable__bpk-mixins/typography';
 
+/* After adding the wrapper around the icon, need to set the line
+    height to stop the 16x16 or 32x32(zoomed) icon from expanding
+    verically past it's 16 or 32 pixels height */
+$icon-height: tokens.$bpk-line-height-xs / 2;
```

Comment:
Given this is only used in one place does this need a whole dedicated variable just for one line further down?

Reply:
Ah, this is because of Styelint, i.e.:
![Screenshot 2024-03-28 at 12 02 28](https://github.com/Skyscanner/backpack/assets/7976511/fee4aace-6afe-4106-9b7a-53a1f14cd744)

It's fine if I am just doing:
```
  &__arrow {
    margin: 0 tokens.bpk-spacing-sm();
    line-height: tokens.$bpk-line-height-xs;    <<--------------
    fill: tokens.$bpk-text-disabled-day;
  }
```

but when I try and divide it by 2, stylelint wants a separate variable.

Reply:
The other option is I can set it to a specific pixel value of no greater than 12 pixels before the alignment starts to break again for the regular and zoomed in icon, e.g.:
`line-height: 12px;`

Or the other option is:
`line-height: 0;` which forces the element to only use the vertical space it needs.

But both will need a stylelint disable.

Or `line-height: 75%`, `line-height: 0.75`, etc.

But again, all will need stylelint disable:

```
42:5   ✖  Expected variable, function or keyword for "0.75" of "line-height"           scale-unlimited/declaration-strict-value
```

Reply:
![Screenshot 2024-03-28 at 12 18 10](https://github.com/Skyscanner/backpack/assets/7976511/a9b49a7a-5cdd-44c7-b078-3c988b1ce89e)

Reply:
Ah yeah, I actually first attached a screenshot with brackets and then I replaced it for one without the brackets:
![Screenshot 2024-03-28 at 14 11 24](https://github.com/Skyscanner/backpack/assets/7976511/eb99a545-cb96-48cb-8755-562af721da85)

Reply:
So with or without brackets produces the same result basically.

Reply:
No worries in which case approved!